pkg/kubelet/prober/results/results_manager.go

```diff
@@ -87,3 +115,26 @@ func (m *manager) Remove(id kubecontainer.ContainerID) {
 	defer m.Unlock()
 	delete(m.cache, id)
 }
+
+func (m *manager) Updates() <-chan Update {
+	updates := m.updates.Load().(chan Update)
```

Comment:
I understand the reason behind using the atomic value here, but was wondering whether we are over-optimizing...Anyway, lgtm.